terraform/environments/portal/rds.tf

```diff
@@ -0,0 +1,300 @@
+locals {
+  # General
+  region = "eu-west-2"
+
+  # RDS
+  igdb_dbname                     = "IGDB"
+  igdb_storage_size               = "200"
+  igdb_auto_minor_version_upgrade = false
+  igdb_backup_retention_period    = "35"
+  igdb_character_set_name         = "AL32UTF8"
+  igdb_instance_class             = "db.t3.large"
+  igdb_engine                     = "oracle-ee"
+  igdb_engine_version             = "19.0.0.0.ru-2021-10.rur-2021-10.r1"
+  igdb_username                   = "admin"
+  igdb_max_allocated_storage      = "3500"
+  igdb_backup_window              = "22:00-01:00"
+  igdb_maintenance_window         = "Mon:01:15-Mon:06:00"
+  igdb_storage_type               = "gp2"
+  igdb_rds_snapshot_name          = "portal-igdb-spike-manual-mp-31052023"
+  igdb_snapshot_arn          = "arn:aws:rds:eu-west-2:${data.aws_caller_identity.current.account_id}:snapshot:${local.application_data.accounts[local.environment].igdb_snapshot_name}"
+  appstream_cidr             = "10.200.32.0/19"
+  cidr_ire_workspace         = "10.200.96.0/19"
+  workspaces_cidr            = "10.200.16.0/20"
+  cp_vpc_cidr                = "172.20.0.0/20"
+  lzprd-vpc                  = "10.205.0.0/20"
+  mp_account_id              = data.aws_caller_identity.current.account_id
+ }
+
+resource "aws_db_subnet_group" "igdb" {
+  name       = "${local.application_name}-${local.environment}-subnetgrp"
+  subnet_ids = [data.aws_subnet.data_subnets_a.id, data.aws_subnet.data_subnets_b.id, data.aws_subnet.data_subnets_c.id]
+
+  tags = merge(
+    local.tags,
+    { "Name" = "${local.application_name}-${local.environment}-subnetgrp" },
+    { "Keep" = "true" }
+
+  )
+
+}
+
+resource "aws_db_parameter_group" "igdb-parametergroup-19c" {
+  name        = "${local.application_name}-${local.environment}-parametergroup"
+  family      = "oracle-ee-19"
+  description = "${local.application_name}-${local.environment}-parametergroup"
+
+
+  parameter {
+    name  = "open_cursors"
+    value = "1000"
+    apply_method = "pending-reboot"
+  }
+
+  parameter {
+    name  = "processes"
+    value = "1000"
+    apply_method = "pending-reboot"
+  }
+
+  parameter {
+    name  = "query_rewrite_enabled"
+    value = "TRUE"
+    apply_method = "pending-reboot"
+  }
+
+  parameter {
+    name  = "query_rewrite_integrity"
+    value = "TRUSTED"
+    apply_method = "pending-reboot"
+  }
+
+  parameter {
+    name  = "sessions"
+    value = "1000"
+    apply_method = "pending-reboot"
+  }
+
+  parameter {
+    name  = "sqlnetora.sqlnet.allowed_logon_version_server"
+    value = "11"
+    apply_method = "pending-reboot"
+  }
+
+  tags = merge(
+    local.tags,
+    { "Name" = "${local.application_name}-${local.environment}-parametergroup" },
+    { "Keep" = "true" }
+  )
+}
+
+resource "aws_db_option_group" "PortalIGDB19OptionGroup" {
```

Comment:
Please amend the Terraform resource local name to standard convention (e.g. igdb)

Reply:
Note that changing this would likely mean TF will recreate this resource, which might result in some dependency issue, so you can leave it for now if preferred.